src/content/pages/capi.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: C API Summit
+subtitle: A summit to bring together users (and interested people) of the C API that Python provide, to discuss the state of this ecosystem, existing challenges, and ongoing work.
+---
+
+# C API Summit
+
+* **When**: Monday, July 8th.
+* **Where**: Prague Congress Centre (PCC), Room Club D
+* **Who can join**: Anyone with a valid in-person EuroPython 2024 ticket - Conference, Tutorial or Combined (see [ticket types](https://ep2024.europython.eu/tickets#ticket-types) for details).
+
+The C API has many different stakeholders, including but not limited to
+CPython core developers, maintainers of Python extensions, code generators
+(Cython, pybindy11, etc) and alternative API design (HPy).
+
+The main goal of this summit is to make sure that people from all these
+different projects can meet and discuss about the state of the C API, existing
+challenges and ongoing work.
+
+## Agenda
+
+* 9:00: Meet and greet - nconference-y post-it based organisation.
```

Comment:
```suggestion
* 9:00: Meet and greet - unconference-y post-it based organisation.
```

I think this is supposed to look like this?

Reply:
yes :)